.github/workflows/ci.yaml

```diff
@@ -23,10 +23,15 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest]
-        python-version: ["3.10", "3.11", "3.12"]
+        python-version: ["3.10", "3.11", "3.12", "3.13"]
```

Comment:
(not blocking) This test matrix is getting a little chonky, maybe consider dropping an intermediate version (3.11 or 3.12)